glide.lock

```diff
@@ -544,7 +544,7 @@ imports:
   - plugin/rewrite
   - router
 - name: github.com/xenolf/lego
-  version: 67c86d860a797ce2483f50d9174d4ed24984bef2
+  version: aa94fb4696349fd6f35168dabe3ab030ddf3e484
```

Comment:
This right here and the GoDaddy DNS provider on line 561 are all I really want to change. Any help accomplishing that would be much appreciate, very much a noob a gliding.